website/static/js/citationList.js

```diff
@@ -14,8 +14,19 @@ var STYLES = {
 
 var ctx = window.contextVars;
 
+var escapeHTML = function(s) {
+    return s.replace(/&/g, '&amp;')
+            .replace(/"/g, '&quot;')
+            .replace(/</g, '&lt;')
+            .replace(/>/g, '&gt;')
+            .replace(/'/g, '&#x27;')
+            .replace(/\//g, '&#x2F;');
+};
+
 var formatCitation = function(style, data, format) {
-    var citeproc = citations.makeCiteproc(style, data, format);
+    var escapedStyle = escapeHTML(style);
```

Comment:
We escape text that will be displayed to the user. Is this a user-facing value, or only something that is used internally?